src/pages/wallets/index.tsx

```diff
@@ -0,0 +1,524 @@
+  />
+)
+const Divider = (props: BoxProps) => (
+  <Box my={16} w="10%" h="0.25rem" bgColor="homeDivider" {...props} />
+)
+
+const CalloutCardContainer = (props: ChildOnlyProp) => (
+  <CardContainer mt={16} {...props} />
+)
+
+const cards = [
+  {
+    emoji: ":dollar:",
+    title: <Translation id="page-wallets:page-wallets-manage-funds" />,
+    description: <Translation id="page-wallets:page-wallets-manage-funds-desc" />,
+  },
+  {
+    emoji: ":frame_with_picture:",
+    title: <Translation id="page-wallets:page-wallets-your-ethereum-account" />,
+    description: <Translation id="page-wallets:page-wallets-your-ethereum-account-desc" />,
+  },
+  {
+    emoji: ":bust_in_silhouette:",
+    title: <Translation id="page-wallets:page-wallets-your-login" />,
+    description: <Translation id="page-wallets:page-wallets-your-login-desc" />,
+  },
+]
+
+const types = [
+  {
+    emoji: ":cd:",
+    description: <Translation id="page-wallets:page-wallets-cd" />,
+  },
+  {
+    emoji: ":mobile_phone:",
+    description: <Translation id="page-wallets:page-wallets-mobile" />,
+  },
+  {
+    emoji: ":globe_with_meridians:",
+    description: <Translation id="page-wallets:page-wallets-web-browser" />,
+  },
+  {
+    emoji: ":globe_with_meridians:",
+    description: <Translation id="page-wallets:page-wallets-web-browser-extension" />,
+  },
+  {
+    emoji: ":desktop_computer:",
+    description: <Translation id="page-wallets:page-wallets-desktop" />,
+  },
+]
+
+const articles = [
+  {
+    title: <Translation id="page-wallets:page-wallets-protecting-yourself" />,
+    description: "MyCrypto",
+    link: "https://support.mycrypto.com/staying-safe/protecting-yourself-and-your-funds",
+  },
+  {
+    title: <Translation id="page-wallets:page-wallets-keys-to-safety" />,
+    description: <Translation id="page-wallets:page-wallets-blog" />,
+    link: "https://www.coinbase.com/learn/crypto-basics/how-to-secure-crypto",
+  },
+  {
+    title: <Translation id="page-wallets:page-wallets-how-to-store" />,
+    description: "ConsenSys",
+    link: "https://media.consensys.net/how-to-store-digital-assets-on-ethereum-a2bfdcf66bd0",
+  },
+]
+
+const guides = [
+  {
+    title: (
+      <Translation id="page-wallets:additional-reading-how-to-create-an-ethereum-account" />
+    ),
+    link: "/guides/how-to-create-an-ethereum-account/",
+  },
+  {
+    title: <Translation id="page-wallets:additional-reading-how-to-use-a-wallet" />,
+    link: "/guides/how-to-use-a-wallet/",
+  },
+]
+
+export const getStaticProps = (async (context) => {
+  const { locale } = context
+  const lastDeployDate = getLastDeployDate()
+
+  // load i18n required namespaces for the given page
+  const requiredNamespaces = getRequiredNamespacesForPage("/wallets")
+
+  return {
+    props: {
+      ...(await serverSideTranslations(locale!, requiredNamespaces)),
+      lastDeployDate,
+      locale,
+    },
+  }
+}) satisfies GetStaticProps<SSRConfig>
+
+const WalletsPage = ({locale}) => {
```

Comment:
Lets infer the props using `InferGetStaticPropsType`

Reply:
Do we need to pass `locale` from `getStaticProps`? `const { locale } = useRouter()` inside the component should do it (seeing this as the predominant pattern throughout the codebase)

If we ditch it, then we shouldn't need to add it to the Props above, and also wouldn't need `InferGetStaticPropsType` since there will be no props.

Reply:
Make sense! I guess `useRouter` also works on server side, no?